code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -241,8 +236,57 @@
 	C.dir = dir
 	qdel(src)
 
+/obj/item/chair/proc/smash(mob/living/user)
+	var/stack_type = initial(origin_type.buildstacktype)
+	if(!stack_type)
+		return
+	var/remaining_mats = initial(origin_type.buildstackamount)
+	remaining_mats-- //Part of the chair was rendered completely unusable. It magically dissapears. Maybe make some dirt?
+	if(remaining_mats)
+		for(var/M=1 to remaining_mats)
+			new stack_type(get_turf(loc))
+	user.unEquip(src,1) //Even NODROP chairs are destroyed.
+	qdel(src)
+
+
+
+
 /obj/item/chair/hit_reaction(mob/living/carbon/human/owner, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
 	if(attack_type == UNARMED_ATTACK && prob(hit_reaction_chance))
 		owner.visible_message("<span class='danger'>[owner] fends off [attack_text] with [src]!</span>")
 		return 1
-	return 0
\ No newline at end of file
+	return 0
+
+/obj/item/chair/afterattack(atom/target, mob/living/carbon/user, proximity)
+	..()
+	if(!proximity)
+		return
+	if(prob(break_chance))
+		user.visible_message("<span class='danger'>[user] smashes \the [src] to pieces against \the [target]</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/C = target
+			if(C.health < (C.maxHealth*0.5))
```

Comment:
They're entirely unneeded for this, you should only use parentheses if you need byond to do operations in a specific order, and you don't need that here.